Lib/test/test_asyncio/test_windows_events.py

```diff
@@ -118,16 +117,16 @@ def test_pipe(self):
 
         clients = []
         for i in range(5):
-            stream = asyncio.Stream(mode=asyncio.StreamMode.READ,
-                                    loop=self.loop, _asyncio_internal=True)
-            protocol = _StreamProtocol(stream,
-                                       loop=self.loop,
-                                       _asyncio_internal=True)
+            stream_reader = asyncio.StreamReader(loop=self.loop,
+                                                 _asyncio_internal=True)
```

Comment:
Probably `_asyncio_internal` is no longer needed.

Reply:
@1st1 It should probably be removed from the `__init__` of StreamReader (and all corresponding references) since it's no longer internal with Stream removed, but it can remain for StreamReaderProtocol since that class is still intended for internal usage. Is this okay?